docs/content/en/docs-dev/user-guide/managing-piped/configuring-notifications.md

```diff
@@ -68,7 +68,7 @@ spec:
           hookURL: https://slack.com/prod
       - name: integration-slack-api
         slack:
-          oauthToken: "token"
+          oauthTokenData: "token"
```

Comment:
I think you should add new field to test it, because oauthToken isn't deleted.

Reply:
done I add more test case and also validate case added